jwql/tests/test_edb_interface.py

```diff
@@ -17,26 +17,70 @@
 
         pytest -s test_edb_interface.py
 """
-from astropy.time import Time
 import pytest
+from astropy.time import Time
+from astroquery.mast import Mast
+
+from jwql.edb.edb_interface import mnemonic_inventory, query_single_mnemonic
+from jwql.utils.utils import get_config
+
+
+@pytest.mark.xfail
+def test_get_mnemonic():
+    """Test the query of a single mnemonic."""
+    from jwql.edb.engineering_database import get_mnemonic
+
+    mnemonic_identifier = 'IMIR_HK_ICE_SEC_VOLT4'
+    start_time = Time('2019-01-16 00:00:00.000', format='iso')
+    end_time = Time('2019-01-16 00:01:00.000', format='iso')
+
+    mnemonic = get_mnemonic(mnemonic_identifier, start_time, end_time)
+    assert len(mnemonic.data) == mnemonic.meta['paging']['rows']
 
 
 @pytest.mark.xfail
+def test_get_mnemonics():
+    """Test the query of a list of mnemonics."""
+    from jwql.edb.engineering_database import get_mnemonics
```

Comment:
This was because upon import `engineering_database` executes `settings = get_config()`, which raised an error in the past (I am not sure if that changed with the recent changes related to config.json). 
I agree that the current implementation is not satisfactory.

Reply:
Understood; the config file certainly make things more complicated. I imagine the recent changes would still raise an error, so best to leave the import statements within the functions.